internal/cloud/backend_test.go

```diff
@@ -303,6 +366,11 @@ func TestCloud_configWithEnvVars(t *testing.T) {
 			if tc.expectedHostname != "" && tc.expectedHostname != b.hostname {
 				t.Fatalf("%s: hostname not valid: %s, expected: %s", name, b.hostname, tc.expectedHostname)
 			}
+
+			if tc.expectedWorkspaceName != "" && tc.expectedWorkspaceName != b.WorkspaceMapping.Name {
+
```

Comment:
An unnecessary blank line which you might delete if already making changes to this file 😜